src/OrbitGl/TextRenderer.h

```diff
@@ -5,102 +5,28 @@
 #ifndef ORBIT_GL_TEXT_RENDERER_H_
 #define ORBIT_GL_TEXT_RENDERER_H_
 
-#include <GteVector.h>
-#include <freetype-gl/mat4.h>
-#include <freetype-gl/texture-atlas.h>
-#include <freetype-gl/texture-font.h>
-#include <freetype-gl/vec234.h>
-#include <glad/glad.h>
-#include <stddef.h>
-#include <stdint.h>
-
-#include <map>
-#include <unordered_map>
-#include <vector>
-
-#include "CoreMath.h"
-#include "PickingManager.h"
-#include "PrimitiveAssembler.h"
 #include "TextRendererInterface.h"
 #include "Viewport.h"
 
-namespace ftgl {
-struct vertex_buffer_t;
-struct texture_font_t;
-}  // namespace ftgl
-
 namespace orbit_gl {
 
 class TextRenderer : public TextRendererInterface {
```

Comment:
Can you explain why this can't be in `OpenGlTextRenderer`. It's not obvious to me why we need the viewport_ (and translation) here since we're not doing anything with it.

Reply:
IMO it makes sense from a perspective of consistency as `BatcherInterface`, `Batcher` and `OpenGlBatcher` have the same distribution of responsibilities across their hierarchy. If this is moved to the OpenGl implementation, the same should happen for the batcher.

Reply:
I split the Interface with the OpenGl implementation, but especially translations there is no place in the interface unless we don't call it Interface. We can discuss about that offline but since it was approved for Batcher, I would prefer following consistency as Florian suggested